specification/cosmos-db/resource-manager/readme.md

```diff
@@ -39,6 +40,16 @@ input-file:
 - Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json
 ```
 
+## Suppression
+``` yaml
+directive:
+  - suppress: TrackedResourceGetOperation

```

Comment:
Suppressions should be more specific. For example, they should specify where in the swagger the supression should be directed. Take a look at [this example](https://github.com/Azure/autorest/tree/master/Samples/2b-suppressions)